addon/components/lazy-image.js

```diff
@@ -6,17 +6,20 @@ var get       = Ember.get;
 var set       = Ember.set;
 var observer  = Ember.observer;
 var Component = Ember.Component;
+var defaultLazyImage = "assets/lz-image/lazy_1x1.gif";
```

Comment:
Yea, I also dislike this approach, but do you have experience with using SVG as a placeholder? How Chrome will react on it? Because when chrome does not have an image for src tag, it will add a gray border around the image, which is kinda sux. 

Or you want to have full SVG placeholder than to remove it and add image?